pkg/operator/workloadcontroller/workload_controller.go

```diff
@@ -276,3 +311,70 @@ func (c *ServiceCatalogAPIServerOperator) namespaceEventHandler() cache.Resource
 		},
 	}
 }
+
+func (c ServiceCatalogAPIServerOperator) deleteOwnerRefFromServiceBindings() error {
+	// Get service bindings
+	gvr := schema.GroupVersionResource{
+		Group:    "servicecatalog.k8s.io",
+		Version:  "v1beta1",
+		Resource: "servicebindings"} // lowercase plural is important
+
+	// get all of the service bindings
+	bindings, err := c.dynamicClient.Resource(gvr).List(metav1.ListOptions{})
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			klog.Info("No service bindings found, nothing to delete.")
+			return nil
+		} else if apierrors.IsServiceUnavailable(err) {
```

Comment:
This worries me. 

Could there be a case where this is offline but we have not deleted the OwnerRef's and this is going to cause errant GC?

Reply:
When I tested this I had deleted the apiservice by hand `oc delete apiservice ...servicecatalog...` and this code returned an error and went into a loop. Because it was not a NOTFOUND error so we returned the err.

I could let it just run and if you want to stop this thing you have to put the apiservice back?

Reply:
I think I need to return any error except notfound.
https://bugzilla.redhat.com/show_bug.cgi?id=1625194#c11

Reply:
basically if it isn't found, it's already gone. Any other error is an error, we need to keep trying. The apiservice should NOT be gone because the operator yas not removed it yet.

Reply:
it is important that we delete the resource so that it is removed from etcd, if the apiservice for that resource is down, then getting stuck in terminating is expected.